src/controllers/planterOrganization.controller.ts

```diff
@@ -123,18 +123,10 @@ export class PlanterOrganizationController {
     @param.path.number('id') id: Number,
     @requestBody() planter: Planter,
   ): Promise<void> {
-    throw new HttpErrors.Unauthorized(
-      'Organizational user has no permission to do this operation',
-    );
-    const result = await this.planterRepository.findById(id);
-    const entityIds = await this.treesRepository.getEntityIdsByOrganizationId(
-      organizationId.valueOf(),
-    );
-    if (!entityIds.includes(result?.organizationId?.valueOf() || -1)) {
+    if (organizationId) {
```

Comment:
We need to fix this properly – either org users should be able to update their own planters' details, or the edit functionality should be disabled for org users in the client and only available to top-level planter managers.

Reply:
New issue once decision is made about direction?